evm_loader/program/src/evm/database.rs

```diff
@@ -274,6 +278,14 @@ mod tests {
             unimplemented!();
         }
 
+        fn rent(&self) -> &Rent {
+            unimplemented!();
```

Comment:
Why `unimplemented!`?

Reply:
This `TestDatabase` is used only for `cargo tests` and `rent()` method isn't required.